05_syntax_lab/01.py

```diff
@@ -1,5 +1,11 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+print "Exercise # 1 "
+
+num = 0
+for i in range(10):
+    print "Number # %d:" % (i+1)
+    tmpnum = int(raw_input())
+    if tmpnum >= num:
+        num = tmpnum
+
+
+print "Highest Number: %d" % num

```

Comment:
1. Better to leave the comment line at the top. That way when you open the script next week you'll still remember what it does 😄 

2. What would be the result if all numbers are negative? Can you make it less surprising?

3. I think prompting for the number would be better without the newline character (in line 5). Can you find a way to print and let the user type in the input in the same line?